apps/condo/domains/ticket/schema/TicketSource.js

```diff
@@ -1,11 +1,14 @@
-const { Text, Select } = require('@keystonejs/fields')
+const { Select } = require('@keystonejs/fields')
 
+const LocalizedText = require('@core/keystone/fields/LocalizedText')
 const { GQLListSchema } = require('@core/keystone/schema')
 const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
 
 const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
 const { COMMON_AND_ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
 
+const { TICKET_SOURCE_TYPES } = require('@condo/domains/ticket/constants/common')
+
 const READ_ONLY_ACCESS = {
```

Comment:
Maybe it's place is in "common" domain?

Reply:
It's ticket source types, don't think common dommain should know something about it.